.github/workflows/WebAutomatedTests.yml

```diff
@@ -42,3 +42,17 @@ jobs:
               run: npm run release -- ${{ steps.variables.outputs.arg }}
             - name: "Executing E2E tests"
               run: npm run test:e2e -- ${{ steps.variables.outputs.arg }}
+            - name: "Fixing files permissions"
+              if: failure()
+              run: |
+                  sudo find ${{ github.workspace}}/packages/pluggableWidgets -type d -exec chmod 755 {} \;
```

Comment:
Why do the directories require execution rights? Isn't chmod 644 not enough for directories as well?

Reply:
In that case I'm applying to different types then 644 only applies for files.